components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/AuthnDataPublisherUtils.java

```diff
@@ -104,15 +104,17 @@ public static String hashString(String value) throws NoSuchAlgorithmException {
     /**
      * Get metadata array for different tenants with tenant domain
      * @param tenantDomain
-     * @return
+     * @param eventType type of the event
+     * @return metadata array
      */
-    public static Object[] getMetaDataArray(String tenantDomain) {
-        Object[] metaData = new Object[1];
+    public static Object[] getMetaDataArray(String tenantDomain, String eventType) {
```

Comment:
The method signature has been changed here. It will break if this public method is used by other components.
Instead, you need to add an overloaded method and depreciate the existing one